realtime/setupSocketIO.js

```diff
@@ -185,7 +185,7 @@ function init(io, redisStore) {
 
     // check if connection using an authorization header
     } else if (socket.handshake.headers.authorization) {
-      jwtUtils.verifyBotToken(socket.handshake.headers.authorization)
+      jwtUtils.verifyBotSocketToken(socket.handshake.headers.authorization)
```

Comment:
Here is there any way you can use verifyToken method ? So I can authorize any token over there. i.e user token

Reply:
not really the verifyToken method is for verifying tokens that come in from requests not from socket connections it requires a request object

Reply:
but you can do something like this verifyToken(socket.handshake) ? right ?

Reply:
doing this refocus will crash due to their not being any session field inside the socket.handshake

Reply:
Can you make one more function to verifySocketToken ? And use that method over there so we can use user socket token as well in future for verifying user as well. This seems hard coded for bot only. @BrandonMurnane  @iamigo

Reply:
i think thats outside the scope of the stories this code was for if we can get this in and another story made for making a generic verifysockettoken we can prioritize that